### PR TITLE
update default value of tidb_analyze_version in new versions

### PR DESCRIPTION
**Pull request by yiwen92**
### First-time contributors' checklist 

- [X] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
update default value of tidb_analyze_version in new versions

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [X] v5.4 (TiDB 5.4 versions)
- [X] v5.3 (TiDB 5.3 versions)
- [X] v5.2 (TiDB 5.2 versions)
- [X] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by yiwen92** [comment]
@hfxsd please review and cherry pick to target versions, thx.

File: statistics.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ aliases: ['/docs-cn/dev/statistics/','/docs-cn/dev/reference/performance/statist
 
 # 统计信息简介
 
-TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `tidb_analyze_version` 用于控制所收集到的统计信息。目前 TiDB 中支持两种统计信息：`tidb_analyze_version = 1` 以及 `tidb_analyze_version = 2`。在 v5.1.0 以前的版本中，该变量的默认值为 `1`。在 v5.1、v5.2、v5.3 中，该变量的默认值为 `2`，作为实验特性启用。
+TiDB 使用统计信息来决定[索引的选择](/choose-index.md)。变量 `tidb_analyze_version` 用于控制所收集到的统计信息。目前 TiDB 中支持两种统计信息：tidb_analyze_version = 1 以及 tidb_analyze_version = 2。在 v5.1.0 以前的版本，该变量的值仅支持为 `1`，在 v5.1.0 版本引入变量的值 `2`。在 v5.1.0 - v5.1.3 以及 v5.2.0 - v5.2.3 中，该变量的默认值为 `2`，其作为实验特性使用（在 v5.1.4 和 v5.2.4 及其后续版本中该默认值又设为 `1`）。在 v5.3.0 及其之后的新版本中，该变量的默认值设为 `2`。
```
Comment (TomShawn) [inline statistics.md:8]: 这里是否考虑用表格来呈现呢？

|  版本  |   默认值   |
|  :-       |      :-         |
|  v4.0.x，v5.0.x  |   该变量值仅支持为 `1` |
|  v5.1.0 - v5.1.3，v5.2.0 - v5.2.3  |  `2`，作为实验特性      |
|   v5.1.4 及之后的 v5.1.x 版本，v5.2.4 及之后的 v5.2.x 版本  |   `1`  |
|   v5.3.x 及后续版本     |   `2`    |

File: system-variables.md [hunk 2]
```diff
@@ -332,7 +332,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`2`
 - 范围：`[1, 2]`
 - 这个变量用于控制 TiDB 收集统计信息的行为。
-- 在 v5.1.0 以前的版本中，该变量的默认值为 `1`。在 v5.1.0 中，该变量的默认值为 `2`，作为实验特性使用，具体可参照[统计信息简介](/statistics.md)文档。
+- 在 v5.1.0 以前的版本，该变量的值仅支持为 `1`，在 v5.1.0 版本引入变量的值 `2`。在 v5.1.0 - v5.1.3 以及 v5.2.0 - v5.2.3 中，该变量的默认值为 `2`，其作为实验特性使用（在 v5.1.4 和 v5.2.4 及其后续版本中该默认值设为 `1`）。在 v5.3.0 及其之后的新版本中，该变量的默认值设为 `2`。具体可参照[统计信息简介](/statistics.md)文档。
```
Comment (TomShawn) [inline system-variables.md:335]: 同上

**Comment by TomShawn** [comment]
本 PR 已被 https://github.com/pingcap/docs-cn/pull/8411 所替代

**Pull request closed without merging**